Install.md

```diff
@@ -1,321 +1,224 @@
+
+Install:
+
 ````
-flask:
-  DEBUG: True
-  LOG_LEVEL: debug
-  PG_HOST: localhost
-  PG_NAME: bozo
-  PG_PORT: 5432
-  PG_USER: myuser
-  PG_PASSWORD: password
-cities:
-  lyon:
-    tablename: lyon
-    extent: [[1843184.223319, 5175387.163907],[1845435.993083, 5177744.541117]]
-    maxtilesize: 2000
-    srs: "EPSG:3946"
-    attributes: []
-    featurespertile: 20
+    git clone https://github.com/AnalyticalGraphicsInc/3d-tiles-samples
+    cd 3d-tiles-samples
+    npm install
+    npm start
 ````
 
-#### Install python3 required dependencies
-Deploy a Python3 [virtual environment](http://python-guide-pt-br.readthedocs.io/en/latest/dev/virtualenvs/) with all the bells and whistles.
-
-**WARNING**: make sure that
- 1. you are located in the `building-server.git` that you cloned
- 1. you are indeed on the `3d-tiles` branch
- 1. that you are using version3 of python and pip (that pip should be pip3!)
- 
- * OSX :
-   ````
-      unset PYTHONPATH 
-      virtualenv -p python3 venv      # Make sure this was installed with pip3
-   ````
- * Ubuntu :
-   ````
-      virtualenv -p /usr/bin/python3 venv      # Make sure this was installed with pip3
-   ````
- * OSX or Ubuntu:
- ````
-     . venv/bin/activate
-     pip install --upgrade setuptools
-     pip install -e .
-     pip install uwsgi
-     pip install lxml     # Note sure this is truly required but it can't hurt
-   ````
-   
-#### Temporary patch for handling gltf and b3dm
-
-**WARNING**: make sure that
- - you are located in the `building-server.git` that you cloned
- - that you are using version3 of python and pip (that pip should be pip3!)
- 
- * Install py3dtiles:
-   ````
-     git clone https://github.com/Oslandia/py3dtiles.git
-     pip install ./py3dtiles/ --upgrade
-     deactivate #since you are done with all necessary installation
-   ````
-   Warning: when doing the `pip install` make sure you are pointing to the `./py3dtiles` local directory as opposed to just `py3dtiles` that will look for some online repository version that won't be correct.
-   
-
-  * go back into building-server.git folder
-  * activate venv : `. venv/bin/activate`
-
-#### Eventually compute the bounding boxes
-Then launch
+The examples tilesets are hosted at `http://localhost:8003/tilesets/`
+
+If you want to run a remote server for streaming a 3d-tiles tileset, you can use
+an Apache server see Step (6) of Install/InstallVilo3D for more informations on 
+how to setup one).
+
+### Install UDV web client and iTowns framework
+
+ Follow the install notes described here:
+ https://github.com/MEPP-team/UDV/blob/master/install.md
+
+This will install UDV and pull the right version of itowns (with temporal extension).
+
+After this installation, if you need to work on itowns you can do it in
+`UDV/UDV-Core/dist/itowns`. If you do so, beware of the remote which is currently
+a fork  (https://github.com/jailln/itowns) of the master (https://github.com/iTowns/itowns).
+
+## Use
```

Comment:
"Usage" or "Using the above installed client/server" would be clearer ?

Reply:
You're right, done.